### DISCUSSION TITLE
Follow up on Drug Strength Bridge2AI Office Hours with respect to https://doi.org/10.1002/pds.5809

### DISCUSSION THREAD

**Discussion by heschmidt04** [chorus-ai/StandardsModule#102] [Results Sharing Questions]
Hoping that the results sharing Discussion is the right place for this. 
The OHDSI Symposium in NJ was great to learn alot about recent work for OHDSI in tools. 

From participating in the weekly OHDSI CDM and Themis Workgroups there was discussion on calculating drug strength; which ties into the follow up on Drugs in Bridge2AI office hours [here](https://drive.google.com/drive/folders/1KknzFtBBeIBxlWrgHCRM2QMx1qEk9RKs) 

In this Github issue [discussion](https://github.com/OHDSI/CommonDataModel/issues/709), Maxim Moinat points to this [PubMed article ](https://onlinelibrary.wiley.com/doi/10.1002/pds.5809) which talks about a study done to normalize drug strength across several participating OMOP instances. 

Who'd be interested in doing a code walk through of https://github.com/oxford-pharmacoepi/DailyDoseRouteValidationStudy?

Because I'm not that facile in R and I'd like to collaborate on ideas that make the data more robust. 
I'd love to do a walk through of the paper to unpack it. 
And an R code review with someone who is comfortable in R for Office Hours, then follow up to see what we have collectively set up for each sites data. 

With the advent of https://github.com/OHDSI/Strategus to replace some functionality in Atlas with version 2.15 or forward, I'd like to discover if we could create a Case Study and prove/disprove the results of the article with our collective data. 

Thoughts for folks? Tagging @jshoughtaling @p-talapova proactively 

Thanks! 

Best regards,
Heidi

**Comment by heschmidt04** [comment]
Forgot to link to the OHDSI forum page discussion coming out of the github issue linked above. The forum thread is [here](https://forums.ohdsi.org/t/drug-dose-fixes-required-to-allow-a-standardized-calculation-call-for-input/22324).

**Comment by jshoughtaling** [comment]
@heschmidt04 - thanks for bringing this up. I'd be happy to lead a code walk through on this topic. Maybe we can schedule it in for an upcoming office hours?

**Reply by heschmidt04** [reply]
I'd like that. I'd like to determine if SME's feel this is worth exploring since N3C isn't an option. Can you sync with Ciera as well so we close the loop on communication? 
Thanks
Heidi

**Comment by heschmidt04** [comment]
Polina mentioned a SQL script (I think) in this follow up office hours [Drug Strength and Calculation](https://drive.google.com/drive/folders/1vQzr-7L-Wd97PwV0Cn4V2_JG-j-i8v-4?usp=drive_link)
